PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/filesystem/VFSSupportStorageType.kt

```diff
@@ -0,0 +1,8 @@
+package com.microsoft.azure.hdinsight.spark.ui.filesystem
+
+enum class VFSSupportStorageType {
+    ADLSGen2,
```

Comment:
We can just enable only `ADLSGen2` now, and the enum class can be moved into class AzureVirtualFileSystem.